src/runner.js

```diff
@@ -1,4 +1,24 @@
- /* istanbul ignore next */
-require('babel-core/register');
+/* istanbul ignore next */
+var fs = require('fs');
+/* istanbul ignore next */
+var path = require('path');
+
+/* istanbul ignore next */
+var babelrc = fs.readFileSync(path.resolve(__dirname, '../.babelrc'));
```

Comment:
In orer to be able to load this package as dependency and call it. 
Can not call from parent package as babel can not locate config.